src/lolex-src.js

```diff
@@ -394,6 +398,12 @@ function hijackMethod(target, method, clock) {
     target[method].clock = clock;
 }
 
+exports.shouldAdvanceTime = false;
```

Comment:
I agree. well that's up to you. I'm not sure if you have any releases planned but we can definitely refactor that part (and possibly others too).
Would be happy to pitch in for this too. Maybe you could point out other parts that need patching too.